api/src/iot/tests/test_parser_ais_masten.py

```diff
@@ -0,0 +1,330 @@
+import pytest
+from django.conf import settings
+
+from iot import import_utils, import_utils_apis, models
+from iot.import_utils import LatLong, PersonData, SensorData
+from iot.serializers import Device2Serializer
+
+
+@pytest.fixture
+def am_data():  # ais_masten
```

Comment:
Ik zie dit soort afkortingen in alle test bestanden. Hier met de commentaar is het duidelijk, maar op de plekken waar deze fixtures worden gebruikt zijn ze een beetje verwarrend. Misschien `api_data` of iets in die trant is iets duidelijker?